modules/sticky-messages/events/messageCreate.js

```diff
@@ -0,0 +1,68 @@
+const { embedTypeV2 } = require('../../../src/functions/helpers');
+const channelData = {};
+
+/**
+ * Deletes the sticky message sent by the bot
+ * @param {Discord.Client} client
+ * @param {Discord.Message} msg
+ */
+async function deleteMessage(client, msg) {
+    let message;
+    message = await msg.channel.messages.fetch(channelData[msg.channel.id].msg).catch(async () => {
+        const msgs = await msg.channel.messages.fetch({limit: 20});
+        message = msgs.find(m => m.author.id === client.user.id);
+    });
+
+    if (message) message.delete().catch(() => {});
+}
+
+/**
+ * Sends the message to the channel
+ * @param {Discord.Client} client
+ * @param {Discord.Message} msg
+ * @param {Object} configMsg The configured message
+ */
+async function sendMessage(client, msg, configMsg) {
+    channelData[msg.channel.id] = {
+        msg: null,
+        timeout: null,
+        time: client.uptime
+    };
+    const sentMessage = await msg.channel.send(await embedTypeV2(configMsg || 'Message not found',
+        {},
+        {},
+        []
```

Comment:
```suggestion
```
These are the default values, I don't see a technical requirement or restriction for the need to re-set them (and potentially overwrite future default value changes). embedTypeV2 has the same API as embedType (and simply calls it with the same values)